workspace.spec

```diff
@@ -572,6 +572,378 @@ module Workspace {
+				"Datacite:IsContinuedBy"
+				"Datacite:IsDerivedFrom"
+				"Datacite:IsDescribedBy"
+				"Datacite:IsDocumentedBy"
+				"Datacite:IsIdenticalTo"
+				"Datacite:IsMetadataFor"
+				"Datacite:IsNewVersionOf"
+				"Datacite:IsOriginalFormOf"
+				"Datacite:IsPartOf"
+				"Datacite:IsPreviousVersionOf"
+				"Datacite:IsPublishedIn"
+				"Datacite:IsReferencedBy"
+				"Datacite:IsRequiredBy"
+				"Datacite:IsReviewedBy"
+				"Datacite:IsSourceOf"
+				"Datacite:IsSupplementTo"
+				"Datacite:IsSupplementedBy"
+				"Datacite:IsVariantFormOf"
+				"Datacite:IsVersionOf"
+				"Datacite:Obsoletes"
+				"Datacite:References"
+				"Datacite:Requires"
+				"Datacite:Reviews"
+				"CrossRef:BasedOnData"
+				"CrossRef:Finances"
+				"CrossRef:HasComment"
+				"CrossRef:HasDerivation"
+				"CrossRef:HasExpression"
+				"CrossRef:HasFormat"
+				"CrossRef:HasManifestation"
+				"CrossRef:HasManuscript"
+				"CrossRef:HasPreprint"
+				"CrossRef:HasRelatedMaterial"
+				"CrossRef:HasReply"
+				"CrossRef:HasReview"
+				"CrossRef:HasTranslation"
+				"CrossRef:IsBasedOn"
+				"CrossRef:IsBasisFor"
+				"CrossRef:IsCommentOn"
+				"CrossRef:IsDataBasisFor"
+				"CrossRef:IsExpressionOf"
+				"CrossRef:IsFinancedBy"
+				"CrossRef:IsFormatOf"
+				"CrossRef:IsManifestationOf"
+				"CrossRef:IsManuscriptOf"
+				"CrossRef:IsPreprintOf"
+				"CrossRef:IsRelatedMaterial"
+				"CrossRef:IsReplacedBy"
+				"CrossRef:IsReplyTo"
+				"CrossRef:IsReviewOf"
+				"CrossRef:IsSameAs"
+				"CrossRef:IsTranslationOf"
+				"CrossRef:Replaces"
+	*/
+	typedef structure {
+		string id;
+		string description;
+		string relationship_type;
+	} PermanentID;
+
+	/* Title
+		Represents the title or name of a resource.
+
+		The 'title_string' field is required; if no value is supplied for 'title_type', it
+		defaults to 'title'. If the 'title_type' is set to 'translated_title', the appropriate
+		BCP-47 tag must be supplied in the 'title_language' field.
+
+		string title_string - the resource title
+		string title_language (optional) - language that the title is in, as a IETF BCP-47 tag.
+			Examples: 'fr'; 'jp-JP'; 'zh-Hant-CN'; 'en-Latn-GB'; 'mn-Cyrl'.
+		string title_type (optional) - the type of title described; must be one of the following:
+			"title" (default)
+			"subtitle"
+			"alternative_title"
+			"translated_title"
+			"other"
+	*/
+	typedef structure {
+		string title_string;
+		string title_language;
+		string title_type;
+	} Title;
+
+	/* Contributor
+
+		Represents a contributor to the resource.
+
+		Contributors must have a 'contributor_type', either 'person' or 'organization'.
+
+		There must also be an identifier or name of some sort for the contributor.
+
+		If the contributor is an organization, either the 'contributor_id' or
+		the 'name' field must be populated.
+
+		An individual can be represented by either 'given_name' and 'family_name'; a 'name'
+		(if the name is not in the binomial format); or a `contributor_id`, such as an
+		ORCID.
```

Comment:
I'm wondering
* why not just use the name field and drop *_name
* if name should be required regardless of whether an ID is available. That'd be better for display purposes anyway, I'd think

Reply:
We discussed this quite a bit in the CE group.

> why not just use the name field and drop *_name

majority preferred to keep the given/family name structure (despite problems inherent in assuming person names conform to this syntax, etc.).

> if name should be required regardless of whether an ID is available.

👍 

> That'd be better for display purposes anyway, I'd think

Ideally at some point -- either in the credit engine or when displaying the contributors -- this data could be dynamically sourced from providers like ORCID.

Reply:
:+1: for requiring the name

I'm still iffy about the two different name fields. Couldn't you split `name` on white space and if there's two parts assume it's family and given?

Reply:
FWIW I agree. I'll bring it up at the CE meeting tomorrow and see if minds can be changed.

Reply:
Elisha is firm on wanting to have given / family names.

Reply:
Now using name / credit_name instead

Reply:
:+1: